### ISSUE TITLE
Merge into W3C repo and archive the WICG repo

### ISSUE THREAD

**Issue by tidoust** [WICG/hdcp-detection#15]
The feature described in this repo is now in scope of the Media Working Group.

Similar to https://github.com/WICG/encrypted-media-encryption-scheme/issues/16, it would be good to merge relevant spec updates back into the main [w3c/encrypted-media](https://github.com/w3c/encrypted-media/) repo (at least as a pull request) so that we can point people at the right place and archive this repository. @joeyparrish, how would you like to proceed?

**Comment by joeyparrish** [comment]
Like the encryption scheme proposal, this proposal has also already been merged into w3c/encrypted-media. If you can point me at some docs or describe the process to me, I can take care of whatever the next steps are for archival and redirection.

**Comment by joeyparrish** [comment]
No, wait, I'm wrong!  I should never trust my memory.  This has **not** been merged back in yet.

**Comment by marcoscaceres** [comment]
Ok, archiving based on the above.
